src/constants/FontSize.ts

```diff
@@ -1,4 +1,5 @@
 export default {
   connectionLabelText: '15px',
   iconSize: 50,
+  configurationsLabel: '25px'
```

Comment:
This font size is too big, in my opinion - you can maybe reduce it to 20px or 15px. Other pages are also showing smaller text and this one is too big, making it difficult to navigate through